pkg/state/state.go

```diff
@@ -42,6 +42,7 @@ const (
 // ReleaseSetSpec is release set spec
 type ReleaseSetSpec struct {
 	DefaultHelmBinary string `yaml:"helmBinary,omitempty"`
+	EnableLiveOutput  *bool  `yaml:"enableLiveOutput,omitempty"`
```

Comment:
I believe this is more like an operational feature than a part of helm releases declared in helmfile.yaml. helmfile.yaml is meant to contain anything for helm releases that are to be templated/diffed/applied/linted by helmfile, but how commands like helm/diff/apply/lint/etc are run.
That said, can we remove this from ReleaseSetSpec and provide it as a command-line flag only, so that it's more consistent with other operational flags?

Reply:
Hi @mumoshu , thanks for the feedback and for taking the time to explain the concept around operational flags, completely understood and definitely make sense, I will address your suggested change shortly and will circle back here once done.